dashboard/app/models/concerns/levels/levels_within_levels.rb

```diff
@@ -73,6 +73,50 @@ def clone_child_levels(parent_level, new_suffix, editor_experiment: nil)
 
         return update_params
       end
+
+      # Create ParentLevelsChildLevel many-to-many relationships based on the
+      # contents of the specified levels' `contained_level_names` or `project_template_level_name` properties.
+      # Differentiated by level_kind (either contained or project template)
+      def setup_child_levels_for(levels, level_kind)
+        outdated_levels_child_level_ids = []
+        new_levels_child_levels = []
+
+        levels.each do |level|
+          # Determine which level names to use based on level_kind
+          level_names = level_kind == ParentLevelsChildLevel::CONTAINED ? level.contained_level_names : [level.project_template_level_name]
```

Comment:
Same as above, I'd love to do something like this instead:
```suggestion
          child_level_names = child_level_kind == ParentLevelsChildLevel::CONTAINED ? level.contained_level_names : [level.project_template_level_name]
```